usecase/parser/msg.go

```diff
@@ -152,6 +152,12 @@ func ParseBlockTxsMsgToCommands(
 func ParseMsgSend(
 	parserParams utils.CosmosParserParams,
 ) ([]command.Command, []string) {
+	// Getting possible signer address from Msg
+	var possibleSignerAddresses []string
+	if parserParams.Msg != nil && parserParams.Msg["from_address"] != nil {
```

Comment:
I am actually wondering if `parserParams.Msg["from_address"]` is `nil`able

Reply:
Thanks for remind